modules/core/src/test/java/org/apache/ignite/marshaller/GridMarshallerAbstractTest.java

```diff
@@ -124,97 +81,14 @@ public abstract class GridMarshallerAbstractTest extends GridCommonAbstractTest
         }
     };
 
-    /** */
-    protected GridMarshallerAbstractTest() {
-        super(/*start grid*/true);
-    }
-
-    /** {@inheritDoc} */
-    @Override protected IgniteConfiguration getConfiguration(String igniteInstanceName) throws Exception {
-        IgniteConfiguration cfg = super.getConfiguration(igniteInstanceName);
-
-        CacheConfiguration namedCache = new CacheConfiguration(DEFAULT_CACHE_NAME);
-
-        namedCache.setName(CACHE_NAME);
-        namedCache.setAtomicityMode(TRANSACTIONAL);
-
-        cfg.setMarshaller(marshaller());
-        cfg.setCacheConfiguration(new CacheConfiguration(DEFAULT_CACHE_NAME), namedCache);
-
-        return cfg;
-    }
-
     /**
      * @return Marshaller.
      */
-    protected abstract Marshaller marshaller();
+    protected abstract Marshaller marshaller() throws IgniteCheckedException;
 
     /** {@inheritDoc} */
     @Override protected void beforeTest() throws Exception {
-        marsh = grid().configuration().getMarshaller();
-        igniteInstanceName = grid().configuration().getIgniteInstanceName();
-    }
-
-    /**
-     * @throws Exception If failed.
-     */
-    @Test
-    public void testDefaultCache() throws Exception {
```

Comment:
Why most of the methods are deleted from this class? These method don't exploit `setMarshaller`, but check ignite system objects marshalling using provided marshaller. Perhaps it still can be used inside ignite somewhere (since OptimizedMarshaller and JdkMarshaller are still used).

Reply:
Correct. Changes reverted.